### PR TITLE
Update access-point.md

### PR DESCRIPTION
**Pull request by SurferTim**
Modified instructions (passphrase length) and divider between the NAT and bridge setup sections. No technical changes made.

### CODE REVIEW

File: configuration/wireless/access-point.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ You need to edit the hostapd configuration file, located at /etc/hostapd/hostapd
 sudo nano /etc/hostapd/hostapd.conf
 ```
 
-Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of NameOfNetwork, and a password AardvarkBadgerHedgehog. Note that the name and password should **not** have quotes around them. 
+Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of NameOfNetwork, and a password AardvarkBadgerHedgehog. Note that the name and password should **not** have quotes around them. The passphrase should be between 8 and 64 charaters in length.
```
Comment (JamesH65) [inline configuration/wireless/access-point.md]: Typo
charaters

File: configuration/wireless/access-point.md [hunk 2]
```diff
@@ -186,15 +187,27 @@ Add the bridging information at the end of the file.
 auto br0
 iface br0 inet manual
 bridge_ports eth0 wlan0
+
 ```    
 
-The access point setup is almost the same as that shown in the previous section. Follow the instructions above to set up the `hostapd.conf` file, but add `bridge=br0` below the `interface=wlan0` line, and remove or comment out the driver line.
+The access point setup is almost the same as that shown in the previous section. Follow the instructions above to set up the `hostapd.conf` file, but add `bridge=br0` below the `interface=wlan0` line, and remove or comment out the driver line. The passphrase must be bewteen 8 and 64 characters long.
```
Comment (JamesH65) [inline configuration/wireless/access-point.md]: Type
bewteen

File: configuration/wireless/access-point.md (hunk 1 above)
Comment (JamesH65) [reply configuration/wireless/access-point.md]: @SurferTim Did you see these comments?

Comment (SurferTim) [reply configuration/wireless/access-point.md]: That is correct. In hostapd.conf, the ssid and passphrase should not have quotes around them. Actually, the passphrase should be no less than 8 characters and no more than 63.

Comment (JamesH65) [reply configuration/wireless/access-point.md]: I was thinking more of the spelling mistakes...

**Comment by JamesH65** [comment]
I've fixed up the typos, and am pushing to full copy edit.

**Pull request merged**